src/Academic/Bundle/BugTrackingBundle/Placeholder/PlaceholderFilter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Academic\Bundle\BugTrackingBundle\Placeholder;
+
+use Symfony\Component\HttpFoundation\Request;
+
+class PlaceholderFilter
+{
+    /**
+     * @var Request
+     */
+    protected $request;
+
+    /**
+     * PlaceholderFilter constructor.
```

Comment:
Please, avoid useless (e.g. autogenerated) comments. Comments should provide valuable information for developer, if they don't do that - it's better to remove them. In this case would be better to have only phpdoc in method comment.